yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -119,19 +131,27 @@ func runExportDataStatusCmdDbzm(streamChanges bool) ([]*exportTableMigStatusOutp
 	var row *exportTableMigStatusOutputRow
 
 	for _, tableStatus := range status.Tables {
-		row = getSnapshotExportStatusRow(&tableStatus)
+		row = getSnapshotExportStatusRow(&tableStatus, leafPartitions, msr)
 		rows = append(rows, row)
 	}
 	return rows, nil
 }
 
-func getSnapshotExportStatusRow(tableStatus *dbzm.TableExportStatus) *exportTableMigStatusOutputRow {
+func getSnapshotExportStatusRow(tableStatus *dbzm.TableExportStatus, leafPartitions map[string][]string, msr *metadb.MigrationStatusRecord) *exportTableMigStatusOutputRow {
 	nt, err := namereg.NameReg.LookupTableName(fmt.Sprintf("%s.%s", tableStatus.SchemaName, tableStatus.TableName))
 	if err != nil {
 		utils.ErrExit("lookup %s in name registry: %v", tableStatus.TableName, err)
 	}
+	displayTableName := nt.ForMinOutput()
+	partitions := leafPartitions[nt.ForOutput()]
```

Comment:
In `getLeafPartitionsFromRootTable` , you are setting the key as `root = rootTable.Qualified.MinQuoted` and logic for `ForOutput` is `t.CurrentName.Qualified.MinQuoted`, that's why this works. 

IMO, this is a bit fragile. Either try to have leafPartitions return a structmap[nametuple] OR let's document this as comments clearly here and in `getLeafPartitionsFromRootTable` as a TODO.

Reply:
> Either try to have leafPartitions return a structmap[nametuple] 

I remember this struct map wasn't helpful in this case as I have to get the NameTuples and create manually in leaf partition cases so to avoid I was using the sqlName in that function. I can add a comment properly for this.